front/app/api/idea_statuses/useUpdateIdeaStatus.ts

```diff
@@ -17,7 +17,7 @@ const updateIdeaStatus = ({ id, requestBody }: IUpdateIdeaStatusObject) =>
   fetcher<IIdeaStatus>({
     path: `/idea_statuses/${id}`,
     action: 'patch',
-    body: requestBody,
+    body: { idea_status: requestBody },
   });
```

Comment:
@IvaKop Here as well - I just don't see it being nested under `idea_status` in the tests for these hooks, so just want to double-check?